docs/guide/extensibility/completions.md

```diff
@@ -186,7 +186,34 @@ even if the prefix only partially matches
 the snippets' tab triggers.
 
 ::: tip Note
-Completions sourced from words in the buffer 
-can be suppressed explicitly 
+Completions sourced from words in the buffer
+can be suppressed explicitly
 from an `on_query_completions` event hook.
 :::
+
+
+## Auto Completion Triggers and Selector
+
+Sublime Text provides two settings
+for users and package authors
+to tweak their auto completion behavior.
+
+- `auto_complete_selector` expects a scope selector
+  that when matched causes the auto completion window
+  to open for any non-punctuation character,
+  i.e. for identifiers.
+  The default configuration disables this
+  for syntaxes with a focus on plain text, comments and strings.
+
+  Before 4070,
+  the selector was applied to the position *after* the just-typed character.
+
+- `auto_complete_triggers` expects a list of mappings

```

Comment:
If the `auto_complete_selector` matches, but none of the `auto_complete_triggers` match, is the AC still shown?

Reply:
They use different logic and only one of them needs to match to trigger the completions panel. I should clarify this.